Handler/Sites.hs

```diff
@@ -22,7 +23,7 @@ getSitesR = do
 
 postSitesR :: Handler Html
 postSitesR = do
-    userId <- requireAuthId
+    Entity userId user <- requireAuth
```

Comment:
Do we want to add a quota check here to prevent race conditions or people hacking the form?